File: scala-gatling/src/gatling/scala/org/openapitools/client/model/ResponseTimeMapResult.scala

package org.openapitools.client.model


case class ResponseTimeMapResult (
    _searchId: String,
    _shapes: List[ResponseShape],
    _properties: ResponseTimeMapProperties
)
object ResponseTimeMapResult {
    def toStringBody(var_searchId: Object, var_shapes: Object, var_properties: Object) =
        s"""
        | {
        | "searchId":$var_searchId,"shapes":$var_shapes,"properties":$var_properties
        | }
        """.stripMargin
}

File: fsharp-functions/OpenAPI/src/model/ResponseTimeMapResult.fs
namespace OpenAPI.Model

open System
open System.Collections.Generic
open Newtonsoft.Json
open OpenAPI.Model.ResponseShape
open OpenAPI.Model.ResponseTimeMapProperties

module ResponseTimeMapResult = 

  //#region ResponseTimeMapResult

  [<CLIMutable>]
  type ResponseTimeMapResult = {
    [<JsonProperty(PropertyName = "search_id")>]
    SearchId : string;
    [<JsonProperty(PropertyName = "shapes")>]
    Shapes : ResponseShape[];
    [<JsonProperty(PropertyName = "properties")>]
    Properties : ResponseTimeMapProperties;
  }
  
  //#endregion
  